backend/src/modules/tournament-tables/core/application/command/AssignTournamentTablesCommandHandler.ts

```diff
@@ -15,9 +16,16 @@ export class AssignTournamentTablesCommandHandler implements CommandHandler<Assi
 
   async execute(command: AssignTournamentTables): Promise<CommandResult> {
     const tournamentId = command.tournamentId;
-    const tournamentTables = await this.repository.findByTournamentId(tournamentId);
+    const tournamentTables = await this.repository.findAllByTournamentId(tournamentId);
+    const tablesToAssign = command.tables.map((table) => {
+      return {
+        tableNumber: TableNumber.from(table.tableNumber),
+        tableName: table.tableName,
+      };
+    });
+    const newCommand = { tournamentId, tables: tablesToAssign };
```

Comment:
Można to nazwać domainCommand - chyba bedzie odpowiednia nazwa :) A te co przychodzą to są ApplicationCommand (ale już nie wprowadzałem tego rozróżnienia w nazwach).

Reply:
Poprawiłam :)